models/ncnews.model.js

```diff
@@ -11,7 +11,7 @@ exports.fetchAllTopics = () => {
 exports.fetchArticleID = (id) => {
   return db
     .query(
-      "SELECT articles.author, articles.title, articles.article_id, articles.body, articles.topic, articles.created_at, articles.votes, COUNT(comments.article_id)::INT AS comment_count FROM articles LEFT JOIN comments ON comments.article_id = articles.article_id WHERE articles.article_id = $1 GROUP BY articles.article_id;",
+      "SELECT users.username AS author, articles.title, articles.article_id, articles.body, articles.topic, articles.created_at, articles.votes, COUNT(comments.article_id)::INT AS comment_count FROM articles LEFT JOIN comments ON comments.article_id = articles.article_id JOIN users ON users.username = articles.author WHERE articles.article_id = $1 GROUP BY articles.article_id, users.username;",
       [id]
     )
```

Comment:
Authors in articles references users.username as a foreign key so you can get the author from the articles tables without joining the users table... I'm sure this works though!